pages/collection.html

```diff
@@ -175,6 +175,12 @@ <h1 id="title-header">The Regis Altare Birthday Project 2023</h1>
           </label>
         </section>
 
+        <div id="collection-foreword" class="hidden">
+          <p><strong>Disclaimer:</strong> The cards in our lovingly fan-curated RegIsekai deck run on pure creativity, and are not regulated or balanced for play. The unplayability is an artistic choice to reflect the chaotic evil of the Demon God King Regis himself, and definitely not because balancing a playable TCG is really, really hard.</p>
+          <p>I mean… you can still <em>try</em> to play it, but a mald-free experience is far from guaranteed.</p>
+          <button id="foreword-close" aria-label="close" type="button">Dismiss</button>
```

Comment:
aria-label isn't needed since the button already has the text Dismiss. aria-label is usually only used for icon buttons w/ no text

Reply:
Thanks, I can remove. I wasn't sure if Dismiss and Close would be considered the same thing to be honest, so I thought I'd just add it for clarity.